app/views/home/index.html.haml

```diff
@@ -1,8 +1,7 @@
 - content_for :header_tags do
-  %link{ href: asset_pack_path('features/getting_started.js'), crossorigin: 'anonymous', rel: 'preload', as: 'script' }/
-  %link{ href: asset_pack_path('features/compose.js'), crossorigin: 'anonymous', rel: 'preload', as: 'script' }/
-  %link{ href: asset_pack_path('features/home_timeline.js'), crossorigin: 'anonymous', rel: 'preload', as: 'script' }/
-  %link{ href: asset_pack_path('features/notifications.js'), crossorigin: 'anonymous', rel: 'preload', as: 'script' }/
+  - if theme_data['preload']
```

Comment:
I'm not sure the value of `theme_data` is guaranteed to respond to `#[]`; maybe we need a default?

(My scenario: If someone is using theme `foo` which is removed by the admin at a later time, it looks like `theme_data` will evaluate to `nil`.)

Reply:
Unless I did something wrong, `theme_data` should always be non-nil because it uses `current_theme`, which falls back to `default` if the theme doesn't exist

Reply:
Ah, so it does.  I guess I missed that.